src/services/addressResolver/index.ts

```diff
@@ -1,12 +1,18 @@
-import axios from "axios";
+import axios, { AxiosAdapter } from "axios";
```

Comment:
why adding a cache ? loading taking long ? people staying more than 5 mins in the website ?

Reply:
lets say that one page is using 3 x "0xabcd...1234", it will call the api 3 times...